libsvmdata/libsvm.py

```diff
@@ -26,7 +26,9 @@
          'sector_test': 'multiclass/sector/sector.t',
          'url': 'binary/url_combined',
          'webspam': 'binary/webspam_wc_normalized_trigram.svm',
-         }
+         'smallNORB': 'multiclass/smallNORB',
+         'mnist': 'multiclass/mnist',
+         'rcv1_multiclass': 'multiclass/rcv1_train.multiclass'}
```

Comment:
now, if they are compressed, you explicitley need the .bz2 extension (ony if they are compressed)